app/controllers/requests_controller.rb

```diff
@@ -20,6 +20,16 @@ def create
       unless @request.save 
         render json: {error: "Request not saved"}, status: :bad_request
       end
+      PrivatePub.publish_to "/valet/new", :request => {id: @request.id,
+                                                name: "#{@request.user.first_name} #{@request.user.last_name}",
+                                                picture: @request.user.profile_picture,
+                                                transmission: @request.user.is_manual,
+                                                phone: @request.user.phone_number,
+                                                latitude: @request.source_location.latitude,
+                                                longitude: @request.source_location.longitude,
+                                                location: @request.source_location.address,
+                                                type: "pick_up"
+                                              }
```

Comment:
How can I render this in jbuilder?   I tried to use render but it spits out an error.  I looked at docs and it told me to do it through a j render on a erb file through a view